pkg/sql/distsql_plan_bulk.go

```diff
@@ -52,3 +52,15 @@ func (dsp *DistSQLPlanner) SetupAllNodesPlanning(
 	})
 	return planCtx, nodes, nil
 }
+
+// GetAllCompatibleNodes returns all nodes that are OK to use in the DistSQL
+// plan.
+func GetAllCompatibleNodes(planCtx *PlanningCtx) []roachpb.NodeID {
```

Comment:
should we just have SetupAllNodesPlanning do this ?

Reply:
yep